cypress/integration/form_spec.js

```diff
@@ -1,19 +1,24 @@
 describe ('Form user flow', () => {
   beforeEach(() => {
     cy.visit('/')
```

Comment:
Added intercepts here because when the App component renders, we trigger a fetch request.